sos/report/plugins/ceph_common.py

```diff
@@ -41,45 +41,52 @@ class Ceph_Common(Plugin, RedHatPlugin, UbuntuPlugin):
 
     # This check will enable the plugin regardless of being
     # containerized or not
-    files = ('/etc/ceph/ceph.conf',)
+    files = ('/etc/ceph/ceph.conf',
+             '/var/snap/microceph/*',)
 
     def setup(self):
         all_logs = self.get_option("all_logs")
 
-        self.add_file_tags({
-            '.*/ceph.conf': 'ceph_conf',
-            '/var/log/ceph(.*)?/ceph.log.*': 'ceph_log',
-        })
-
-        if not all_logs:
-            self.add_copy_spec("/var/log/calamari/*.log",)
+        if not self.path_exists('/var/snap/microceph/common'):
```

Comment:
ack this can be done and in fact was thinking about this approach in the last part of https://github.com/sosreport/sos/pull/3291#issuecomment-1617435936 , I will change it to this approach